openfe/tests/protocols/test_openmmutils.py

```diff
@@ -931,7 +931,7 @@ def test_openeye_import_error(self, monkeypatch, uncharged_mol):
 @pytest.mark.slow
 @pytest.mark.download
 # Sometimes we get a DOI lookup error from duecredit
-@pytest.mark.flaky(reruns=3, only_rerun=ValueError, reruns_delay=10)
+#@pytest.mark.flaky(reruns=3, only_rerun=ValueError, reruns_delay=10)
```

Comment:
It would be good to not leave this lying around long term. @mikemhenry or @atravitz could you open an issue to track the removal (or re-addition) of this flaky mark?

Reply:
sure https://github.com/OpenFreeEnergy/openfe/issues/1161